internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -169,3 +216,34 @@ func parseExpose(line string) []portConfig {
 	}
 	return ports
 }
+
+func parseHealthCheck(line string) (healthCheck, error) {
+	var hc healthCheck
+
+	if line[12:] == "NONE" {
+		return hc, nil
+	}
+
+	var retries int
+	var interval, timeout, startPeriod time.Duration
+	fs := flag.NewFlagSet("flags", flag.ContinueOnError)
+
+	// Default value DurationVar gives is in nanoseconds
+	fs.DurationVar(&interval, "interval", 30*nanoseconds, "")
```

Comment:
Maybe use the existing time const in `time` package instead of defining it in this file?